operator/v1/types_ingress.go

```diff
@@ -513,6 +520,61 @@ type IngressControllerLogging struct {
 	Access *AccessLogging `json:"access,omitempty"`
 }
 
+// IngressControllerHTTPHeaderPolicy is a policy for setting HTTP headers.
+//
+// +kubebuilder:validation:Enum=Append;Replace;IfNone;Never
+type IngressControllerHTTPHeaderPolicy string
```

Comment:
Deviating from the enhancement proposal, I changed the allowed values here from lower-case (`append;replace;if-none;never`) to the above value for consistency with other API values.